src/skim.rs

```diff
@@ -0,0 +1,35 @@
+use crate::configuration::ConfigurationStore;
+use crate::error::Error;
+use anyhow::{Context, Result};
+use skim::prelude::*;
+use std::io::Cursor;
+
+/// Use built-in fuzzy finder to select configuration to activate
+pub fn fuzzy_find_config() -> Result<String> {
+    // load list of configurations into string seperated by newlines
+    let store = ConfigurationStore::with_default_location().context("Opening configuration store")?;
+    let configs_as_str = store
+        .configurations()
+        .iter()
+        .map(|x| x.name())
+        .collect::<Vec<&str>>()
+        .join("\n");
+
+    let options = SkimOptionsBuilder::default()
+        .height(Some("50%"))
+        .build()
+        .or(Err(Error::SkimBuildError))?;
+
+    let item_reader = SkimItemReader::default();
+    let items = item_reader.of_bufread(Cursor::new(configs_as_str));
```

Comment:
I wonder if we can build the items directly given we've already got all the individual strings.

Effectively we create a newline-separated string and then pass this to the item builder which splits on newline again to generate the items. Hopefully we can shortcut that.